### PR TITLE
ICMSLST-2513 Data Migration Timezone Corrections

### PR DESCRIPTION
**Pull request by mikeylockwood**

### CODE REVIEW

File: data_migration/management/commands/post_migration.py [hunk 1]
```diff
@@ -140,12 +140,21 @@ def assign_org_permissions(self, group_name: str, rows: list[tuple[str, str, int
...
+                try:
+                    org = Importer.objects.get(pk=org_id)
+                except Importer.DoesNotExist:
+                    raise Importer.DoesNotExist(f"Importer with pk {org_id} does not exist")
             else:
-                org = Exporter.objects.get(pk=org_id)
+                try:
+                    org = Exporter.objects.get(pk=org_id)
+                except Exporter.DoesNotExist:
+                    raise Exporter.DoesNotExist(f"Exporter with pk {org_id} does not exist")
```
Comment (MattHolmes123) [inline data_migration/management/commands/post_migration.py:157]: This just an unrelated fix?

File: data_migration/utils/format.py [hunk 2]
```diff
@@ -108,12 +109,9 @@ def adjust_icms_v1_datetime(dt_val: dt.datetime) -> dt.datetime:
     # ICMS V1 datetime values are created using this:
     # https://docs.oracle.com/database/121/SQLRF/functions207.htm#SQLRF06124
     # Therefore replace the naive datetime with the correct timezone
-    aware_dt = dt_val.replace(tzinfo=UK_TZ)
+    aware_dt = dt_val.replace(tzinfo=dt.UTC)
```
Comment (MattHolmes123) [inline data_migration/utils/format.py:112]: Is this comment still correct?

**Review by MattHolmes123 — APPROVED** [review]

File: data_migration/management/commands/post_migration.py (hunk 1 above)
Comment (mikeylockwood) [reply data_migration/management/commands/post_migration.py:157]: Yes unrelated to the current ticket, I just needed it to check some missing users when running

File: data_migration/utils/format.py (hunk 2 above)
Comment (mikeylockwood) [reply data_migration/utils/format.py]: No, removed
